sdk/src/feature_set.rs

```diff
@@ -291,6 +291,10 @@ pub mod max_tx_account_locks {
     solana_sdk::declare_id!("CBkDroRDqm8HwHe6ak9cguPjUomrASEkfmxEaZ5CNNxz");
 }
 
+pub mod cache_failed_tx_executors {
```

Comment:
What's the motivation to gate this change?  Perf?

Reply:
Maybe there isn't one more than habit, I'll remove